### DISCUSSION TITLE
Spirit Tracks Randomizer?

### DISCUSSION THREAD

**Discussion by Midrule** [phst-randomizer/ph-randomizer#353] [Ideas]
I know it's an entirely different game, but one doesn't exist at all for it. It takes a lot of work so it's just an out there request. I didn't know who else to ask so I put it here lol, but it would be cool if it was something in the future.

**Comment by lunarsoap5** [comment]
It's something we have discussed and will probably be something we look into tackling once the PH Rando is in a state we are comfortable with.
